.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: python
+python:
+  - "2.7"
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install mpi-default-bin mpi-default-dev
+
+  # LAMMPS
+  #  - build lammps as shared library
+  #  - and install LAMMPS python wrapper
+  - git clone git://git.lammps.org/lammps-ro.git mylammps
+  # checkout a recent stable version (from 28 June 2014)
+  - cd mylammps/src && git checkout r12164
+  # make shared library for python
+  - make makeshlib && make -f Makefile.shlib ubuntu_simple
+  - cd ../python && sudo python install.py /usr/lib/ $VIRTUAL_ENV/lib/python2.7/site-packages/
+  # make lammps executable and add to path
+  - cd ../src
+  - make ubuntu_simple
+  - ln -s lmp_ubuntu_simple lammps
+  - export PATH=$TRAVIS_BUILD_DIR/mylammps/src/:$PATH
+  - cd $TRAVIS_BUILD_DIR
+
+  # use current master of simphony
+  - pip install git+git://github.com/simphony/simphony-common.git#egg=simphony
```

Comment:
@itziakos   As you have seen in this development work, I am using some new CUBA enums (e.g. CUBA.PAIR_POTENTIALS) during this development.  Later once, we are sure that these are the CUBA keywords, we will submit a PR in simphony-common to update simphony/core/cuba.yml.   

In the meantime,  how should we integrate these new CUBA keywords for the CI.  I thought of either (1) forking simphony-common and adding the enums i need for testing there or (2) in the travis script here, check out simphony-common and then edit the cuba.py (or later the cuba.yml) file before installing.  

Any advice would be appreciated.